.idea/codeStyles/Project.xml

```diff
@@ -3,31 +3,6 @@
     <JetCodeStyleSettings>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
-    <Objective-C-extensions>
```

Comment:
I'd recommend adding `/.idea` to your .gitignore so you don't have to worry about unrelated changes making their way into your PR 🕵🏻‍♂️

Reply:
True I usually have to do that at work. Forgot to do it here. I use Webstorm almost exclusively